zipline/errors.py

```diff
@@ -315,6 +315,33 @@ class MultipleSymbolsFound(ZiplineError):
     """.strip()
 
 
+class MultipleSymbolsFoundForFuzzySymbol(ZiplineError):
```

Comment:
Could a case be made for having this subclass `MultipleSymbolsFound`? I'm not sure from this PR what the motivation for separating these is.

Reply:
the motivation is the error message being different, so I'm not sure if there is a benefit to subclassing `MultipleSymbolsFound`

Reply:
Yeah, one thing I was thinking of is that if any consumer is catching this, the behavior there will change. Probably not a huge deal though.

Reply:
ah that's true, my initial thought was to update consumers that catch this with the more specific exceptions, but they are all the same family of error. Updated  `SameSymbolUsedAcrossCountries` to also be subclassed from `MultipleSymbolsFound`